cmd/gosky/debug.go

```diff
@@ -254,6 +261,172 @@ var debugStreamCmd = &cli.Command{
 	},
 }
 
+var compareStreamsCmd = &cli.Command{
```

Comment:
seems like some other stuff got rolled up in this PR; not reviewing these bits

Reply:
yeah, got pulled in unintentionally, not a big deal but just ignore for this PRs purposes